virtcontainers/hyperstart_agent.go

```diff
@@ -1005,3 +1005,8 @@ func (h *hyper) setGuestDateTime(time.Time) error {
 	// hyperstart-agent does not support setGuestDateTime
 	return nil
 }
+
+func (h *hyper) copyFile(src, dst string) error {
+	// hyperstart-agent does not support copyFile
+	return nil
```

Comment:
Shouldn't this be...

```suggestion
	return errors.New("hyperstart-agent does not support copyFile")
```

Reply:
based on other implementations https://github.com/kata-containers/runtime/blob/master/virtcontainers/hyperstart_agent.go#L999-L1007

I don't think so